doc/api/n-api.md

```diff
@@ -121,6 +121,24 @@ must use Node-API exclusively by restricting itself to using
 and by checking, for all external libraries that it uses, that the external
 library makes ABI stability guarantees similar to Node-API.
 
+### Enum values in ABI stability
```

Comment:
Should we also augment the https://github.com/nodejs/node/blob/main/doc/contributing/adding-new-napi-api.md document along with it?

Reply:
That document is specifically about adding a new node-api function. I think it is not necessary to be verbose on extending existing functions.